# The effect of non-coding variants on transcription factor binding sites

## Prerequisites
All software and packages were run on macOS Sierra Version 10.12.4. 

* Perl Version 5.24.1: https://www.perl.org/get.html
* Python Version 2.7: https://www.python.org/download/releases/2.7/
* MEME Version 4.11.2_2: http://meme-suite.org/doc/download.html
* JAMM Version 1.0.7.5: https://github.com/mahmoudibrahim/JAMM
* Bedtools Version 2.26.0: http://bedtools.readthedocs.io/en/latest/content/installation.html
* Samtools Version: 0.1.20: http://samtools.sourceforge.net/

## Motif Analysis
<pre>
Follow README_dnase.txt to process DNase-seq data.
Follow README_chip.txt to process ChIP-seq data.
The latter will include information about how to run MEME to get predicted binding motifs.
</pre>

## Allelic Imbalance Analysis
<pre>
Follow README_allelic_imbalance.txt.
You can find examples in allelic_imbalance_table.xlsx.
</pre>

## Genome Annotation
<pre>
This was done using DeepSea: http://deepsea.princeton.edu/job/sequence/create/
Refer to DeepSEA_ouptut/DeepSEA_Annotations.pdf for specific chromosomal loci used as input.
</pre>
